task/git-clone/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "example of pre-apply-task-hook, this script can be used to run kubernetes commands"
```

Comment:
since we had only one example test for now, I think this might help people to understand how the `pre-apply-task-hook.sh` script might look like, it is kind of placeholder for now, I guess we can remove it in future when we have some tests which will contain `pre-apply-task-hook.sh`. WDYT?